index.html

```diff
@@ -27,6 +27,39 @@
     <div id="text"></div>
     <div><br /><button id="restart">RESTART</button></div>
     <footer>
+      <br />
```

Comment:
usually the footer would be for things that belong at the very bottom of a page
I would probably move all this form stuff/score stuff above the footer in it's own `<div>`